systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -652,12 +651,24 @@ void testMirrorMaker2CorrectlyMirrorsHeaders() {
 
         //deploying example clients for checking if mm2 will mirror messages with headers
 
-        KafkaBasicClientResource targetKafkaClientsJob = new KafkaBasicClientResource("", targetConsumerName,
-            KafkaResources.plainBootstrapAddress(kafkaClusterTargetName), targetExampleTopic, messageCount, "", ClientUtils.generateRandomConsumerGroup(), 1000);
+        KafkaBasicExampleClients targetKafkaClientsJob = new KafkaBasicExampleClients.Builder()
+            .withConsumerName(targetConsumerName)
+            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(kafkaClusterTargetName))
+            .withTopicName(targetExampleTopic)
+            .withMessageCount(MESSAGE_COUNT)
+            .withDelayMs(0)
```

Comment:
Here will be `0`? Not the `1000`?

Reply:
Good catch, that's rebasing fault :smile: